pontoon/base/models.py

```diff
@@ -1867,7 +1861,7 @@ def checkout_path(self):
     @property
     def can_commit(self):
         """True if we can commit strings back to this repo."""
-        return self.type in ("svn", "git", "hg")
+        return self.type in (self.Type.SVN, self.Type.GIT, self.Type.HG)
```

Comment:
We should drop the `can_commit` logic entirely (can be done in a followup), because this will always be True.

Reply:
Sure, I can do this in a follow-up PR.

Reply:
Thanks, filed https://bugzilla.mozilla.org/show_bug.cgi?id=1686278.